src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -73,6 +73,9 @@ class WorkspaceTabs extends PureComponent {
       ])
     }
     const isOwner = workspace && Utils.isOwner(workspace.accessLevel)
+    const iconHelp = (iconName, iconLabel) => {
+      return h(Fragment, [icon(iconName, { size: 15, style: { marginRight: '.5rem' } }), iconLabel])
```

Comment:
This abstraction is a good idea, since it cleans up duplication on the icons. However, having the label in here seems extraneous, since it's just concatenating them. What about renaming the helper as `menuIcon`, have it just return the icon, and have the labels just sit next to them in the DOM below.